wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # if template is incomplete locally, delete and ask for a download
+    if not phishingpage.check_template(TEMPLATE_NAME):
```

Comment:
I kindly disagree with that we should only run them for some templates. So here is what I propose:

1. All templates have their own dictionary. For example :
```python
MINIMAL = {"index.html": "http://pastebin.com/raw/qCfpF8Jx",
           "bg.jpg": "http://i.imgur.com/moAqxKR.jpg",
           "loading.gif": "http://i.imgur.com/7Jv1O2D.gifv",
           "logo.png": "http://i.imgur.com/6u75LmV.png",
           "masthead.jpg": "http://i.imgur.com/C9KesWR.jpg",
           "style.css": "http://pastebin.com/raw/fQATivV7",
           "upgrading.html": "http://pastebin.com/raw/gaUJXhML"}
```
Advantages of using this method is that if any of the files ever gets deleted or corrupted, it will automatically install new ones.
2. Only include ```minimal``` in the github repository.
Over 90% of the users currently use the ```minimal``` template and not the other templates. It also saves a little bit of space. This problem would be solved when I make an interactive template chooser. I already have most of the work done. A big disadvantage of going with you method will be that ```check_template``` will no longer be a generic function and I feel it would be a dirty code.